doc/source/user_guide/numpy_images.rst

```diff
@@ -184,29 +184,28 @@ argument.
 
 Many functions in ``scikit-image`` can operate on 3D images directly::
 
+    >>> import numpy as np
+    >>> import scipy as sp
```

Comment:
Hmm, this made me look up: https://stackoverflow.com/questions/36014733/official-abbreviation-for-import-scipy-as-sp-sc

Reply:
This advice comes from before lazy loading was added to SciPy, so `import scipy as sp` should now be a perfectly acceptable way to do it.

Reply:
`import scipy as sp`

Reply:
Ok, thanks (I guess it echoes `import numpy as np`).

Reply:
Yeah, definitely. I have never really seen `sc` in the wild and I like how
- **N**um**P**y > `np`
- **S**ci**P**y > `sp`
- **s**ci**k**it-**i**mage > `ski`
consistent this is.

Reply:
It was also the official recommendation for many years, but this was removed between
- https://docs.scipy.org/doc/scipy-0.14.0/reference/tutorial/general.html
- https://docs.scipy.org/doc/scipy-0.15.0/reference/tutorial/general.html

I believe I added it in 2008 (when we were trying to unify the calling conventions)
- https://github.com/scipy/scipy/commit/3f374927ec7b7bd4392a41040c28b450a2db434d
and it was removed in 2014
- https://github.com/scipy/scipy/commit/ba9cf21dba6f169535f41406215deb17da16c7e9

I am happy to see it coming back!!

Reply:
@jarrodmillman good to know, thanks for sharing!